doc/interfaces/types.Referrer.md

```diff
@@ -0,0 +1,17 @@
+# Interface: Referrer
+
+[types](../modules/types.md).Referrer
+
+Referrer object to be used in maker order API
+
+## Properties
+
+### address
+
+• **address**: `string`
+
+___
+
+### rate
+
+• **rate**: `BigNumberish`
```

Comment:
Yeah. ethers v6 itself provides the union type called `BigNumberish`, which includes string, number and bigint. So it's better to still call it `BigNumberish`, instead of just `bigint`